### PR TITLE
Update Finnish translation

### PR DESCRIPTION
**Pull request by Kuujaku**

### CODE REVIEW

File: po/fi.po [hunk 1]
```diff
@@ -225,105 +244,447 @@ msgstr "asp palautti {}"
...
+msgid "If no arguments are provided 'paru -Syu' will be performed"
+msgstr "Jos argumenttejä ei ole annettu 'paru -Suy' suoritetaan"
+
+#: src/help.rs:26
+msgid "Options without operation:"
+msgstr "Valinnat ilman toimintoja:"
+
+#: src/help.rs:27
+msgid "    -c --clean            Remove unneeded dependencies"
+msgstr "	-c --clean            Poista käyttämättömät riippuvuudet"
```
Comment (Morganamilo) [inline po/fi.po:331]: Alignment broken

File: po/fi.po [hunk 2]
```diff
@@ -225,105 +244,447 @@ msgstr "asp palautti {}"
...
+msgstr "    --[no]upgrademenu      Näytä vuorovaikutteinen valikko ohittaaksesi päivitykset"
+
+#: src/help.rs:58
+#, fuzzy
+msgid "    --[no]removemake       Remove makedepends after install"
+msgstr "    --[no]removemake       Poista make riippuvuudet asennuksen jälkeen"
+
+#: src/help.rs:59
+msgid "    --[no]cleanafter       Remove package sources after install"
+msgstr "	--[no]cleanafter       Poista paketin lähdekoodit asennuksen jälkeen"
```
Comment (Morganamilo) [inline po/fi.po:448]: Alignment broken

File: po/fi.po [hunk 3]
```diff
@@ -225,105 +244,447 @@ msgstr "asp palautti {}"
...
+
+#: src/help.rs:79
+msgid ""
+"    --topdown              Shows repository's packages first and then AUR's"
+msgstr ""
+"    --topdown              Näyttää pakettivaraston paketit ensin sitten AURin"
+
+#: src/help.rs:81
+msgid "show specific options:"
+msgstr "näytä spesifiset valinnat:"
```
Comment (Morganamilo) [inline po/fi.po:561]: show is the argument name

File: po/fi.po [hunk 4]
```diff
@@ -225,105 +244,447 @@ msgstr "asp palautti {}"
...
+msgid "Usage:"
+msgstr "Käyttö:"
+
+#: src/help.rs:5
+msgid "    paru"
+msgstr "    paru"
+
+#: src/help.rs:6
+msgid "    paru <operation> [...]"
+msgstr "    paru <operation> [...]"
```
Comment (Morganamilo) [inline po/fi.po:263]: Most things in <..> are untranslated.

**Comment by Morganamilo** [comment]
From what I see the alignment is still totally broken.

**Comment by Kuujaku** [comment]
I'm sorry this consumed more of your time than what this translation is worth.

**Pull request closed without merging**